test/functionalTest/cases/2750_common_metrics/2750_totalPayloadAsMetric.test

```diff
@@ -30,17 +30,19 @@ brokerStart CB
 --SHELL--
 
 #
-# 01. Send a GET /version
+# 01. Send a GET /v2/entities
 # 02. Ask for Metrics, see 1 Incoming Transaction, but nothing about incomingTransactionRequestSize
 # 03. Create an entity (with payload of exactly 50 bytes)
 # 04. Ask for Metrics, see 4 Incoming Transactions, and 50 bytes in incomingTransactionRequestSize
 # 05. Create an entity (with payload of exactly 100 bytes)
 # 06. Ask for Metrics, see 6 Incoming Transactions, and 150 bytes in incomingTransactionRequestSize
+# 07. GET /v2/entities - to provoke payload in response
+# 08. Ask for Metrics, see 8, and XXX bytes as incomingTransactionResponseSize
```

Comment:
I guess your plan was to replace XXX by an actual bytes number. I think is not a good idea, as this may change from time to time and, while we can fix in the test step itself, maybe we forget in the preamble. So better to use neutral messages for preamble/echos.

(Maybe the same problem happens in other .test that I haven't realized)